### PR TITLE
Aaron B's POC and Vagrant, Docker work

### PR DESCRIPTION
**Pull request by birkland**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,98 @@
...
+* For Vagrant users, this is pre-configured to be `10.11.8.11`
+* For Docker Compose users:
+	* For Linux duning services directly on a local docker, this is `localhost`
+	* For Docker Machine users, you can find this by running `docker-machine ip apix`
+
+Next, verify that some services are running
+* Verify Fedora is running. Point your browser to `http://${ADDR}:8080/rest`
+	* You should see Fedora, with an empty root container
+	* Create an object through Fedora's UI
+* point your browser to Fuseki at `http://${ADDR}:3030/fcrepo`
```
Comment (awoods) [inline README.md]: Should this instead be:
`http://${ADDR}:3030/fuseki` ?

File: Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    m.vm.provider :docker do |d|
+      d.name = 'indexing-triplestore'
+      d.build_dir = "proof-of-concept/indexing-triplestore"
+      d.build_args = ["-t", "apix-poc/indexing-triplestore"]
+      d.volumes = ["/shared:/shared"]
+      d.remains_running = true
+      d.env = {
+        FCREPO_BASEURL: "fcrepo:8080/rest",
+        JMS_BROKERURL: "tcp://fcrepo:61616",
+        TRIPLESTORE_BASEURL: "fuseki:3030/fcrepo/update",
```
Comment (whikloj) [inline Vagrantfile]: Following on from @awoods [comment](https://github.com/fcrepo4-labs/fcrepo-api-x/pull/1/files#r61341266) should this be `fuseki:3030/fuseki/update`??

File: docker/builder/out [hunk 3]
```diff
@@ -0,0 +1,420 @@
+Apache Maven 3.2.5 (12a6b3acb947671f09b81f49094c53f426d8cea1; 2014-12-14T12:29:23-05:00)
```
Comment (whikloj) [inline docker/builder/out]: Did you want to checkin this log?

File: README.md (hunk 1 above)
Comment (acoburn) [inline README.md]: No, the `DEFAULT_FUSEKI_DATASET` is "fcrepo". This is correct, as I understand it.

File: Vagrantfile (hunk 2 above)
Comment (acoburn) [inline Vagrantfile]: Nope, see above. This is correct.

File: README.md (hunk 1 above)
Comment (ajs6f) [inline README.md]: That might be better changed to "fedora_triple_index" or something like that. Or maybe the confusion is just inevitable.

Comment (awoods) [inline README.md]: :+1: @ajs6f, avoiding confusion seems like a good goal.

Comment (birkland) [inline README.md]: Good point, will change the name

File: docker/builder/out (hunk 3 above)
Comment (birkland) [inline docker/builder/out]: It looks like this was included by error, it shouldn't be there.  Thanks!

**Comment by acoburn** [comment]
The commits in this PR were merged by @emetsger

**Pull request closed without merging**